packages/quantic/force-app/main/default/lwc/quanticNotifications/__tests__/quanticNotifications.test.js

```diff
@@ -97,6 +113,12 @@ function mockErroneousHeadlessInitialization() {
   };
 }
 
+// Simulates a search status update
+function mockSearchStatusUpdate() {
```

Comment:
Either you put this directly in your test, or you find it a better name, in the context we have when reading that function name, we have no idea what you're updating the searchStatus to.

`mockSearchStatusHasResults` for example describes what you're doing now.

Otherwise I would simply put the code within your tests specifically since what you update the state to can be very specific to the test you're running.

Reply:
will change naming